com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueProcessor.cs

```diff
@@ -85,7 +94,7 @@ public void ProcessReceiveQueue(NetworkUpdateStage currentStage)
         /// ProcessSendQueue
         /// Called to send both performance RPC and internal messages and then flush the outbound queue
         /// </summary>
-        public void ProcessSendQueue()
+        internal void ProcessSendQueue()
```

Comment:
why all these `public` to `internal` changes? the class itself is already `internal`, there is no extra protection coming from all these `public` to `internal` changes anyway?